BlocSpot/BlocSpot/LocationManager.m

```diff
@@ -0,0 +1,33 @@
+//
+//  LocationManager.m
+//  BlocSpot
+//
+//  Created by McCay Barnes on 8/6/15.
+//  Copyright (c) 2015 McCay Barnes. All rights reserved.
+//
+
+#import "LocationManager.h"
+
+
+@implementation LocationManager
+
+
++ (instancetype) sharedLocationManager{
+    static dispatch_once_t once;
+    static id sharedLocationManager;
+    dispatch_once(&once, ^{
+        sharedLocationManager = [[self alloc] init];
+    });
+    return sharedLocationManager;
+}
```

Comment:
Does the singleton look correct? I copied it from the Blocstagram app checkpoint.